src/state.js

```diff
@@ -34,12 +35,16 @@ module.exports = class MonoplasmaState {
         this.members = initialMembers.map(m => new MonoplasmaMember(undefined, m.address, m.earnings))
         /** @property {MerkleTree} tree The MerkleTree for calculating the hashes */
         this.tree = new MerkleTree(this.members)
-
+        /** @property {string}  adminAddress the owner address who receives the admin fee and the default payee if no memebers */
+        this.adminAddress  = adminAddress
+        /** @property {number}  adminFeeFraction fraction of revenue that goes to admin */
+        this.setAdminFeeFraction(adminFeeFraction)
+        
         this.indexOf = {}
         this.members.forEach((m, i) => { this.indexOf[m.address] = i })
 
         // add default address in case revenue is received with no members
-        const defaultAddress = isAddress(defaultReceiverAddress) ? defaultReceiverAddress : "0x0000000000000000000000000000000000000000"
+        const defaultAddress = isAddress(adminAddress) ? adminAddress : "0x0000000000000000000000000000000000000000"
```

Comment:
bad address should be an error, it's really not great to burn the "extra" tokens by default, that's why there is an admin address for fallback

Reply:
ok